Source/src/modules/ModuleCamera.cpp

```diff
@@ -66,7 +80,9 @@ void Hachiko::ModuleCamera::Controller(const float delta) const
 #ifndef PLAY_BUILD
     if (!App->editor->GetSceneWindow()->IsHovering())
         return;
-#endif
+#endif  
+    if (main_camera->GetCameraType() != ComponentCamera::CameraType::GOD)
+        return;
```

Comment:
Remember use of braces even for one line if content

Reply:
I saw this in the code many times thats why I also used it, if we all agree to use the braces always then perfect!